back/lib/shared/hello/hello.go

```diff
@@ -0,0 +1,13 @@
+package hello
+
+import (
+	"strings"
+)
+
+func HelloWorld() (string, int) {
+	return "HelloWorld", 25
+}
+
+func returnLastname(fullName string) string {
+	return strings.Split(fullName, " ")[1]
+}
```

Comment:
Could you remove this file in another commit as it's content is on the readme
But thx for providing test sample

Reply:
Same as below, I'll remove it in the next PR when I add other real tests cases.

Reply:
Ok